esp/esp/program/tests.py

```diff
@@ -121,9 +175,12 @@ def testUserSearchFn(self):
         self.assertStringContains(response['location'], "/manage/userview?username=testuser123543")
 
         # Full name, unique
-        response = c.get("/manage/usersearch", { "userstr": "Admin User" })
```

Comment:
Never mind, I didn't notice that it was hit several lines below in the unchanged part.